### ISSUE TITLE
CSV Import with booleans.

### ISSUE THREAD

**Issue by TindSolutions** [FlutterFlow/flutterflow-issues#877]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Importing in Firebase manager

Uploading a CSV containing booleans. Preview and everything looks great, but the boolean is imported as a string.

Please see https://community.flutterflow.io/c/discuss-and-get-help/importing-bools-causing-display-issues

### Expected Behavior

Imported as a boolean.

### Steps to Reproduce

Import a CSV file.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

none

### Context

importing data

### Visual documentation

https://community.flutterflow.io/c/discuss-and-get-help/importing-bools-causing-display-issues

### Environment

```markdown
- FlutterFlow version: 
- Platform:
- Browser name and version: chrome
- Operating system and version affected:  windows
```

**Comment by bulgariamitko** [comment]
This will be a hard one as boolean can be represented by different ways like TRUE/true/1 and so on. What is your current data? I will mark this as won't fix it for that reason. I made a great video how you can import csv manually which I think will fix your issue 
https://www.youtube.com/watch?v=4Ir1j6L48Lg&t=9s

**Comment by TindSolutions** [comment]
Did you read the community posts at all? After reviewing my files I was asked by your chat-support to add this bug report as he agreed it was a bug. I have tried all the variant possible. True, TRUE, true, 1. I had to abandon FF and do the import with other tools, as other people also seem to be doing.

![Skjermbilde 2023-05-13 191918](https://github.com/FlutterFlow/flutterflow-issues/assets/133334611/7ca2c402-40ab-4687-a03d-6538622d28f6)
